vite.config.ts

```diff
@@ -2,10 +2,11 @@ import { fileURLToPath, URL } from 'node:url'
 
```

Comment:
Please, remove this empty line to trigger the sonar scan again

Reply:
@FelixSiegfriedRiedel ping

Reply:
Sure, I'm onto it!